README.md

```diff
@@ -2,3 +2,71 @@ FIG Cookies
 ===========
 
 Managing Cookies for PSR-7 Requests and Responses.
+
+# Accessing cookies from Request
+
+```php
+use Dflydev\FigCookies\Cookies;
+
+// $request = a psr7request;
+$cookies = Cookies::fromRequest($request);
+```
+
+## Check a cookie exists
+
+You can check whether a cookie by name exists via `has` method which returns `bool`.
+
+```php
+$cookies->has('name');
+```
+
+## Get cookie
+
+You can get a `Cookie` object via `get` method.
+
+```php
+$cookie = $cookies->get('name');
+```
+
+This will return a `Dflydev\FigCookies\Cookie` object via which you can get
+name, value etc via `getName()` and `getValue()` respectively.
+
+## Change cookie value
+
+You can change the value of a cookie via `withValue` method.
+
+```php
+$cookie->withValue('some value');
+```
+
+or alternatively you can create a fresh cookie object via
+
+```php
+use Dflydev\FigCookies\Cookie;
+
+$cookie = new Cookie('name');
+// or
+// $cookie = Cookie::create('name');
+
+$cookie->withValue('some value');
+```
+
+## Setting cookie to header
```

Comment:
This section is wrong; I'm working on finishing the API for this. Right now it is going to be:

```php

// $response = a psr7response

// Create a SetCookies collection from a response.
$setCookies = SetCookies::fromResponse($response);

// Create a SetCookie.
SetCookie::create('lu')
    ->withValue('Rg3vHJZnehYLjVg7qi3bZjzg')
    ->withExpires('Tue, 15-Jan-2013 21:47:38 GMT')
    ->withMaxAge(500)
    ->withPath('/')
    ->withDomain('.example.com')
    ->withSecure(true)
    ->withHttpOnly(true)
;

// Set the SetCookie in the SetCookies collection.
$setCookies = $setCookies->with($setCookie);

// Render the SetCookies collection into the response.
$response = $setCookies->renderIntoSetCookieHeader($response);
```